### ISSUE TITLE
ClayNavigation.Breadcrumb High Level Component

### ISSUE THREAD

**Issue by diegonvs** [liferay/clay#2140]
Sample API:

```jsx
<ClayNavigation.Breadcrumb maximumItems={3}>
	<ClayLink href=“/home”>Home</ClayLink>
	<ClayLink href=“/about”>About</ClayLink>
	<ClayLink href=“/projects”>Projects</ClayLink>
	<ClayLink href=“/contact”>Contact</ClayLink>
</ClayNavigation.BreadCrumb>
```

**Comment by bryceosterhaus** [comment]
What do you think about something like...

```jsx
<ClayNavigation.Breadcrumb
     items={[
           {href: '/home', name: 'Home'},
           {href: '/about', name: 'About'}
     ]}
     buffer={3} /> // similar to pagination
/>
```

**Comment by diegonvs** [comment]
> What do you think about something like...
> 
> ```js-jsx
> <ClayNavigation.Breadcrumb
>      items={[
>            {href: '/home', name: 'Home'},
>            {href: '/about', name: 'About'}
>      ]}
>      buffer={3} /> // similar to pagination
> />
> ```

For high level makes sense.

What do you think about to receive property for `items` and also enable composition as I mentioned?

**Comment by diegonvs** [comment]
I agree with changing from `maximumItems` to `buffer`

**Comment by bryceosterhaus** [comment]
I'm not sure composition is super necessary here since its so simple and we are already composing from <Link />. You could always write it in the same way you would write a low-level, but then just add small wrapper as the main export that would be the high-level version.

**Comment by matuzalemsteles** [comment]
Fixed #2159
